corehq/apps/hqwebapp/static/hqwebapp/js/knockout_bindings.ko.js

```diff
@@ -667,6 +667,8 @@ hqDefine("hqwebapp/js/knockout_bindings.ko", [
         init: function (element, valueAccessor) {
             var list = valueAccessor();
             $(element).sortable({
+                handle: '.grip',
```

Comment:
I also tested the visit scheduler and case list explorer. Each already had an icon with grip. This does change those so that their sortables can only be moved with the arrow. Before the entire row was able to be drag and dropped

Reply:
So earlier one could click anywhere on the row and move it but now they would need to specifically click on the arrow to move the entire row. That might have been added for convenience, not sure though what is a better UX. Though i think this change is better and more explicit.

Reply:
New behavior sounds reasonable.